src/components/OverviewApp/Notary/LargeRequestTable.tsx

```diff
@@ -61,16 +63,36 @@ type LargeRequestTableProps = {
     setSelectedLargeClientRequests: any
     dataForLargeRequestTable: any
     largeRequestListLoading: boolean
+    setDataForLargeRequestTable: any
 }
 
-const LargeRequestTable = ({ setSelectedLargeClientRequests, largeRequestListLoading, dataForLargeRequestTable }: LargeRequestTableProps) => {
+const LargeRequestTable = ({ setSelectedLargeClientRequests, largeRequestListLoading, dataForLargeRequestTable, setDataForLargeRequestTable }: LargeRequestTableProps) => {
 
     const context = useContext(Data)
 
     const CANT_SIGN_MESSAGE = "You can currently only approve the allocation requests associated with the multisig organization you signed in with. Signing proposals for additional DataCap allocations will require you to sign in again";
 
+    const [searched, setSearched] = useState(false)
+
+    useEffect(() => {
+        if (context.searchString) {
+            setDataForLargeRequestTable((prev: any) => prev.filter((item: any) => item.searchBy.toLowerCase().includes(context.searchString.toLowerCase())))
+            setSearched(true)
+        }
+        if (context.searchString === "" && context.largeClientRequests) {
+            const data = context.largeClientRequests
+                .map((item: any) => ({ ...item, data: item.data.name, searchBy: `${item.data.name} ${item.issue_number} ${item.address}` }))
+                .map((item: any) => item.tx !== null ? item : { ...item, tx: "", })
+            setDataForLargeRequestTable(data)
+        }
+    }, [context.searchString, context.largeClientRequests])
+
+
     return (
         <div className="large-request-table" style={{ minHeight: "500px" }}>
+            {!context.ldnRequestsLoading && <p style={{ margin: "0.8rem  1.2rem", color: "#373D3F" }}>
+                * <i style={{ textDecoration: "underline", textUnderlineOffset: "4px" }}>You can look for specific deals querying by "Client", "Address" and "Audit Trail" fields using the search box.</i>
```

Comment:
maybe you cna change to
`Use the search box to search for issues, then  press enter or click on the lens`

Reply:
thanks i did the changes you asked for